src/js/Scroll.js

```diff
@@ -5,25 +5,46 @@ smoothscroll.polyfill();
 
 class Scroll {
   constructor() {
-    // value in pixels elements scrolled to will be from top of the window
-    this.scrollOffset = -60;
+    this.scrollOffset = 0;
 
-    $(document).ready(this.onReady.bind(this));
+    this.isClearHash = true;
+    this.clearHashTimeout = 0;
+
+    this.$header = $('#header');
 
+    $(document).ready(this.onReady.bind(this));
```

Comment:
Just a general comment here. `bind()` returns a new function. In this case `ready` only happens once but repeating this pattern in other events such as `scroll` or `click`, which normally happen more often, presents a performance issue because we are creating a new function each time. 

It's better to bind the function outside, before binding them to the event. That way, the vound function is only created once.